requirements/common.txt

```diff
@@ -168,11 +168,6 @@ python-twitter==3.3
 # To parse po files
 polib==1.0.8
 
-# Needed for sprite generation
-# We're using this version from git to ensure compatibility with
-# Jinja2==2.8 (see https://github.com/jorgebastida/glue/pull/211).
-git+https://github.com/lorenzogil/glue@01c00cd33b9b78ea868300c266c16acd59a81bfc#egg=glue==0.11.1
```

Comment:
@hackerkid next time, please explain why the commit is correct in the commit message.  In this case, it's correct because `glue` is no longer used with the new sprite sheets.